src/os/state/v4/timestate.js

```diff
@@ -520,3 +530,57 @@ os.state.v4.TimeState.prototype.parsePeriod = function(period) {
 
   return null;
 };
+
+/**
+ * Returns slice intervals element for timeranges.
+ * @param {Array<goog.math.Range>} sliceRanges
+ * @return {!Element}
+ * @private
+ */
+os.state.v4.TimeState.prototype.sliceRangesToXml_ = function(sliceRanges) {
+  let slices = os.xml.createElement(os.state.v4.TimeTag.SLICES);
```

Comment:
Please use `var` instead of `let`. The compiler will replace these with `var`, but we haven't officially started allowing ES6 that can be transpiled by Closure. This is in part to ensure we can still load the debug build in older supported browsers, but also for the sake of consistency across the code base.

Reply:
OK. Changed to using `var`.